### PR TITLE
Add Wordpress container hacofile and criu-based config

### PR DESCRIPTION
**Pull request by udzura**
This is rough implementation...

### CODE REVIEW

File: provision/provisioner.sh [hunk 1]
```diff
@@ -54,3 +54,5 @@ apt install -y postfix
 
 # netdata
 bash <(curl -Ss https://my-netdata.io/kickstart.sh) all --non-interactive
+
+tar -xf /var/lib/haconiwa/images/ssh.image.tar -C /var/lib/haconiwa/rootfs/nginx-10-0-5-10/
```
Comment (linyows) [inline provision/provisioner.sh]: The extract of image is done at the initial setup of the container in dispatcher. What does this mean?

https://github.com/FastContainer/nginx-haconiwa/pull/16/files#diff-92007c9f8d9ac747304ec558f0648086L133

File: provision/hacos/wordpress.haco [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Haconiwa.define do |c|
+  id = ENV['ID'] || "wordpress-10-0-5-20"
+  ip = ENV['IP'] || "10.0.5.20"
+  port = ENV['PORT'] || "80"
+
+  c.name = id
+  c.workdir = '/root'
+
+  root = Pathname.new("/var/lib/haconiwa/rootfs/wordpress-10-0-5-20")
```
Comment (linyows) [inline provision/hacos/wordpress.haco]:
```suggestion
  root = Pathname.new("/var/lib/haconiwa/rootfs/#{id}")
```

How about excluding dependencies?

File: provision/nginx/conf.d/dispatcher.rb [hunk 3]
```diff
@@ -104,8 +111,9 @@ def initialize(ip, port, haco, env = [], hostname = '')
       @haco = haco
 
       @root = '/var/lib/haconiwa'
-      @id = "#{@haco}-#{@ip.gsub('.', '-')}"
-      @environment = ["IP=#{@ip}", "PORT=#{@port}", "ID=#{@id}"]
+      @id = "#{@haco}-#{@ip.gsub('.', '-')}-#{Time.now.to_i}"
+      # @id = "#{@haco}-#{@ip.gsub('.', '-')}-1540895210"
+      @environment = ["IP=#{@ip}", "PORT=#{@port}", "ID=#{@id}", "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"]
```
Comment (linyows) [inline provision/nginx/conf.d/dispatcher.rb]: I see.

Comment (udzura) [reply provision/nginx/conf.d/dispatcher.rb]: This `PATH` will be removed when this is supported in haconiwa side.

**Pull request merged**